src/rust/ide/view/src/debug_scenes/vcs.rs

```diff
@@ -0,0 +1,390 @@
+#![allow(missing_docs)]
```

Comment:
This file contains much duplication of interface.rs debug scene doesn't it? It may be frustrating to fix identical code when for example the API of GraphEditor will change. We could put common parts to a separate module, or merge this example with interface.rs, and display the VCS information on some shortcut. Personally I prefer the second solution: adding this shortcut should not obfuscate the example, and it may be convenient to have such features in one place.

Reply:
I second that. I'd definitely merge it with interface.rs! There is no point in having 2 interface testing scenes. Interface.rs should be just a debu scene for ALL interface elements.